gef.py

```diff
@@ -11853,6 +11853,18 @@ def target_remote_posthook():
         except gdb.error:
             pass
 
+    # set fallback 'debug-file-directory' for gdbs that installed outside `/usr`.
+    try:
+        default_dbgsym_path = "/usr/lib/debug"
+        param_name = "debug-file-directory"
+        dbgsym_paths = gdb.parameter(param_name)
+        if default_dbgsym_path not in dbgsym_paths:
+            newpath = f"{dbgsym_paths}:" if dbgsym_paths else ""
+            newpath += default_dbgsym_path
+            gdb.execute(f"set {param_name} {newpath}")
+    except gdb.error:
+        pass
```

Comment:
Avoid silent exceptions

```suggestion
    except gdb.error as e:
        warn(f"Failed to set {param_name}, reason: {str(e)}")
```